sklearn/metrics/classification.py

```diff
@@ -391,72 +391,114 @@ def jaccard_similarity_score(y_true, y_pred, normalize=True,
     y_pred : 1d array-like, or label indicator array / sparse matrix
         Predicted labels, as returned by a classifier.
 
-    normalize : bool, optional (default=True)
-        If ``False``, return the sum of the Jaccard similarity coefficient
-        over the sample set. Otherwise, return the average of Jaccard
-        similarity coefficient.
-
     sample_weight : array-like of shape = [n_samples], optional
         Sample weights.
 
+    average : string, [None (default), 'micro', 'macro', 'weighted']
+        If ``None``, the scores for each class are returned. Otherwise, this
+        determines the type of averaging performed on the data:
+
+        ``'micro'``:
```

Comment:
I think you also need the 'binary' case as in prf. (I know it seems convoluted, but it makes sure users get what they expect)

Reply:
The current multilabel implementation corresponds to prf's average='samples' which should be supported and should remain default.